pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -742,6 +742,7 @@ func Convert_v1_BlockVolumeSource_To_api_Disk(volumeName string, disk *api.Disk,
 	if !contains(volumesDiscardIgnore, volumeName) {
 		disk.Driver.Discard = "unmap"
 	}
+	disk.Source.Name = volumeName
```

Comment:
why is setting the source name required now when we didn't need to set it in the past?

Reply:
It is used in this function to discover which volumes were block devices - https://github.com/kubevirt/kubevirt/blob/bfcd542c4226c3720cc1a8059af109c6474e137a/pkg/ephemeral-disk-utils/utils.go#L108